libafl_targets/src/libfuzzer.rs

```diff
@@ -2,14 +2,54 @@
 //! This makes `LibAFL` interoperable with harnesses written for other fuzzers like `Libfuzzer` and [`AFLplusplus`](aflplus.plus).
 //! We will interact with a C++ target, so use external c functionality
 
-use alloc::{string::String, vec::Vec};
+use alloc::{
+    boxed::Box,
+    rc::{Rc, Weak},
+    string::String,
+    vec::Vec,
+};
+use core::{
+    cell::RefCell,
+    marker::PhantomData,
+    ops::{Deref, DerefMut},
+};
+
+use libafl::{
+    bolts::AsSlice,
+    inputs::{BytesInput, HasBytesVec, HasTargetBytes, UsesInput},
+    mutators::{MutationResult, Mutator},
+    prelude::Rand,
+    state::{HasMaxSize, HasRand},
+    Error,
+};
+use once_cell::unsync::OnceCell;
```

Comment:
Never used OnceCell, but at some point I looked into lazy_static and found out https://crates.io/crates/static_init would be faster than that. Not sure if it applies here